model/src/test/kotlin/utils/LicenseResolverTest.kt

```diff
@@ -1,653 +0,0 @@
-                packages = (analyzer!!.result.packages + pkg.toCuratedPackage()).toSortedSet()
-            )
-        )
-    )
-
-private fun OrtResult.addPathExclude(pathExclude: PathExclude): OrtResult {
-    val repositoryConfig = repository.config.run {
-        copy(
-            excludes = excludes.copy(
-                paths = excludes.paths + pathExclude
-            )
-        )
-    }
-
-    return replaceConfig(repositoryConfig)
-}
-
-private fun OrtResult.addProject(project: Project): OrtResult =
-    copy(
-        analyzer = analyzer!!.copy(
-            result = analyzer!!.result.copy(
-                projects = (analyzer!!.result.projects + project).toSortedSet()
-            )
-        )
-    )
-
-private fun OrtResult.addScanResult(id: Identifier, scanResult: ScanResult): OrtResult {
-    val scanResults = scanner!!.results.scanResults
-    val scanResultsForId = scanResults.find { it.id == id }?.let {
-        scanResults.remove(it)
-        it.results
-    }.orEmpty()
-
-    scanResults += ScanResultContainer(id, scanResultsForId + scanResult)
-
-    return copy(
-        scanner = scanner!!.copy(
-            results = scanner!!.results.copy(
-                scanResults = scanResults
-            )
-        )
-    )
-}
-
-private fun OrtResult.getVcsProvenance(id: Identifier): Provenance? =
-    getProject(id)?.let { Provenance(vcsInfo = it.vcsProcessed) }
-        ?: getPackage(id)?.let { Provenance(vcsInfo = it.pkg.vcsProcessed) }
-
-private fun OrtResult.getSourceArtifactProvenance(id: Identifier): Provenance? =
-    getPackage(id)?.let { Provenance(sourceArtifact = it.pkg.sourceArtifact) }
-
-private enum class ProvenanceType {
-    VCS,
-    SOURCE_ARTIFACT
-}
-
-private fun OrtResult.getProvenance(id: Identifier, type: ProvenanceType): Provenance? =
-    when (type) {
-        ProvenanceType.SOURCE_ARTIFACT -> getSourceArtifactProvenance(id)
-        else -> getVcsProvenance(id)
-    }
-
-private fun Package.createPackageConfig(type: ProvenanceType) =
-    PackageConfiguration(
-        id = id,
-        sourceArtifactUrl = sourceArtifact.url.takeIf { type == ProvenanceType.SOURCE_ARTIFACT },
-        vcs = VcsMatcher(
-            type = vcs.type,
-            url = vcs.url,
-            revision = vcs.revision
-        ).takeIf { type == ProvenanceType.VCS }
-    )
-
-private fun createEmptyOrtResult() =
-    OrtResult(
-        Repository.EMPTY,
-        AnalyzerRun(
-            environment = Environment(),
-            config = AnalyzerConfiguration(ignoreToolVersions = true, allowDynamicVersions = true),
-            result = AnalyzerResult.EMPTY
-        ),
-        ScannerRun(
-            environment = Environment(),
-            config = ScannerConfiguration(),
-            results = ScanRecord(
-                scanResults = sortedSetOf(),
-                storageStats = AccessStatistics()
-            )
-        )
-    )
-
-class LicenseResolverTest : WordSpec() {
```

Comment:
Do we have the tests still somewhere "duplicated" ?

Reply:
Not exactly the same, but the `LicenseInfoResolver` is thoroughly tested.

Reply:
I was trying to find tests which cover the cases tested here, but to me it is not obvious whether `LicenseInfoResolverTest` accomplishes that. For that reason I believe it's not a good idea to delete these. But it might just be me because I wrote these (and found several bugs while writing them). So not blocking this one.